src/.clang-tidy

```diff
@@ -2,6 +2,13 @@ Checks: '
 -*,
 bugprone-argument-comment,
 modernize-use-nullptr,
+performance-*,
+-performance-avoid-endl,
+-performance-enum-size,
+-performance-inefficient-string-concatenation,
+-performance-no-int-to-ptr,
+-performance-noexcept-move-constructor,
+-performance-unnecessary-value-param,
```

Comment:
don't add all of these new performance items to the clang tidy. Only add the `performance-enum-size`